### PR TITLE
Add constants for usage events

### PR DESCRIPTION
**Pull request by d-william**
Motivation:

Facilitate the reading of usage events when consume the usage address.

### CODE REVIEW

**Review by tsegismont — COMMENTED** [review]
Thank you @d-william 

Why prefix with `USAGE_`? `EVENT_` seems enough to me.
Also, can you update other places in the code where the values are used?

**Comment by d-william** [comment]
I had originally put `USAGE_` to avoid confusion with the "announce" events (`ServiceDiscoveryOptions#.getAnnounceAddress();`).

As you requested, I used the constants at the other places.
I only see usages in `DiscoveryImplTestBase` tests, but may be I forget some places?

**Review by tsegismont — APPROVED** [review]
Thank you @d-william , waiting for CI build before merging

**Pull request merged**
